UndertaleModLib/Models/UndertaleSprite.cs

```diff
@@ -697,8 +697,12 @@ public static uint UnserializeChildObjectCount(UndertaleReader reader)
         reader.Position += 4; // "Name"
         uint width = reader.ReadUInt32();
         uint height = reader.ReadUInt32();
+        int marginLeft = reader.ReadInt32();
+        int marginRight = reader.ReadInt32();
+        int marginBottom = reader.ReadInt32();
+        int marginTop = reader.ReadInt32();
```

Comment:
Were these fields always there? why did we never read them before?

Reply:
These are required in 2024.6 and above to determine the size of a sprite's collision mask data, which now depends on the size of its bounding box (which is what these values are).